autoload/terminal_images.vim

```diff
@@ -654,3 +654,26 @@ function! terminal_images#ToggleGlobal() abort
         call terminal_images#EnableGlobal()
     endif
 endfun
+
+function! terminal_images#EnableBuffer() abort
+    let b:terminal_images_auto = 1
+    call terminal_images#ShowAllImages({})
+    echom "Automatic image display is on for this buffer"
+endfun
+
+function! terminal_images#DisableBuffer() abort
+    let b:terminal_images_auto = 0
+    call terminal_images#ClearVisibleImages()
+    echom "Automatic image display is off for this buffer"
+endfun
```

Comment:
Could you add a command for this function and an example in the readme file?

Reply:
Done